src/napari/layers/points/points.py

```diff
@@ -1818,67 +1811,6 @@ def get_ray_intersections(
         )
         return start_point, end_point
 
-    def _set_view_slice(self) -> None:
-        """Sets the view given the indices to slice with."""
-
-        # The new slicing code makes a request from the existing state and
-        # executes the request on the calling thread directly.
-        # For async slicing, the calling thread will not be the main thread.
-        request = self._make_slice_request_internal(
-            self._slice_input, self._data_slice
-        )
-        response = request()
-        self._update_slice_response(response)
-
-    def _make_slice_request(self, dims: 'Dims') -> _PointSliceRequest:
-        """Make a Points slice request based on the given dims and these data."""
-        slice_input = self._make_slice_input(dims)
-        # See Image._make_slice_request to understand why we evaluate this here
-        # instead of using `self._data_slice`.
-        data_slice = slice_input.data_slice(self._data_to_world.inverse)
-        return self._make_slice_request_internal(slice_input, data_slice)
-
-    def _make_slice_request_internal(
-        self, slice_input: _SliceInput, data_slice: _ThickNDSlice
-    ) -> _PointSliceRequest:
-        return _PointSliceRequest(
-            slice_input=slice_input,
-            data=self.data,
-            data_slice=data_slice,
-            projection_mode=self.projection_mode,
-            out_of_slice_display=self.out_of_slice_display,
-            size=self.size,
-        )
-
-    def _update_slice_response(self, response: _PointSliceResponse) -> None:
-        """Handle a slicing response."""
-        self._slice_input = response.slice_input
-        indices = response.indices
-        scale = response.scale
-
-        # Update the _view_size_scale in accordance to the self._indices_view setter.
-        # If out_of_slice_display is False, scale is a number and not an array.
-        # Therefore we have an additional if statement checking for
-        # self._view_size_scale being an integer.
-        if not isinstance(scale, np.ndarray):
-            self._view_size_scale = scale
-        elif len(self._shown) == 0:
-            self._view_size_scale = np.empty(0, int)
-        else:
-            self._view_size_scale = scale[self.shown[indices]]
-
-        self._indices_view = np.array(indices, dtype=int)
-        # get the selected points that are in view
-        self._selected_view = list(
-            np.intersect1d(
-                np.array(list(self._selected_data)),
-                self._indices_view,
-                return_indices=True,
-            )[2]
-        )
-        with self.events.highlight.blocker():
-            self._set_highlight(force=True)
-
```

Comment:
This was removed but is not present in the copy-pasted code below. Is that intentional?